sonar-text-dotnet/pipeline_scripts/build_and_restore_solution.ps1

```diff
@@ -17,3 +17,7 @@ dotnet build `
      /p:CommitId=$env:CIRRUS_CHANGE_IN_REPO `
      /p:BranchName=$env:CIRRUS_BRANCH `
      /p:BuildNumber=$env:BUILD_NUMBER
+
+if ($LASTEXITCODE -eq 1) {
```

Comment:
Any better way to pull the MSBuild exit code?

Reply:
This is how it looks now when it fails:
https://cirrus-ci.com/task/4720181440675840